requirements/rocm.txt

```diff
@@ -0,0 +1,17 @@
+# Common dependencies
+-r common.txt
+
+--extra-index-url https://download.pytorch.org/whl/rocm6.4.1
```

Comment:
Hi @billishyahao - this URL gives me a HTTP 403 (Forbidden).

Since the PyTorch 2.7.x wheels are built with ROCm 6.3, should this be `https://download.pytorch.org/whl/rocm6.3` instead?